var/spack/repos/builtin/packages/py-pyspy/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import sys
+
+from spack.package import *
+
+
+class PyPyspy(PythonPackage):
+    """py-spy is a sampling profiler for Python programs."""
+
+    homepage = "https://github.com/benfred/py-spy"
+    url = "https://files.pythonhosted.org/packages/51/70/e8bfeeac2c57f3af9988ca6d6d12cff3a9a1394e0d9a2c63f08555c0552f/py_spy-0.1.0-py2.py3-none-manylinux1_x86_64.whl"
```

Comment:
We don't add wheels to Spack but try to build from source if possible, see <https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#downloading>.
Can you change that?